agent-side-integrations/crm-helpdesk-integrations/freshdesk.md

```diff
@@ -30,16 +30,42 @@ Install the Freshdesk or Freshchat app from the app gallery (Admin -> Settings -
 
 Add your licence key and private key when prompted (include the first and final lines that look like  `-----BEGIN/END PRIVATE KEY-----`). Cobrowse.io API can be left as-is, unless you are using an account on our EU Cloud or a self-hosted instance.&#x20;
 
-<figure><img src="../../.gitbook/assets/Screenshot 2022-11-23 at 14.04.57.png" alt=""><figcaption><p>Add your licence key and private key in the boxes. </p></figcaption></figure>
+<figure><img src="../../.gitbook/assets/Screenshot 2024-02-16 at 10.24.55.png" alt=""><figcaption><p>Add your licence key and private key in the boxes. </p></figcaption></figure>
 
 Afterwards, click install for the app to install. You can check successful installation by going to Manage Apps in the Admin settings to see if Cobrowse.io is there and enabled.
 
-<figure><img src="../../.gitbook/assets/Screenshot 2022-11-23 at 14.09.40.png" alt=""><figcaption><p>The app is installed and enabled. </p></figcaption></figure>
+<figure><img src="../../.gitbook/assets/Screenshot 2024-02-16 at 10.35.31.png" alt=""><figcaption><p>The app is installed and enabled. </p></figcaption></figure>
 
 The app should then appear in the sidebar during Freshchats and Freshdesk tickets and contacts. If you do not see it then please refresh the page (you may need to log out/in again).
 
 To connect to user devices, your next step is to install our SDKs for your platforms. You can find the links for each platform here: [https://docs.cobrowse.io/](https://docs.cobrowse.io/).
 
+### Group based access control
+
+You can restrict access to Cobrowse.io based on user groups. To do this, navigate to the settings page and select the **Show advanced settings checkbox**. This action will display additional configuration options. By entering the appropriate Freshdesk or Freshchat API credentials, a multi-select input will appear. This input allows you to specify which groups can access Cobrowse.io. If no groups are selected, Cobrowse.io will be accessible to all groups.
```

Comment:
Is it obvious the _settings page_ in the second sentence is the Freshdesk settings page or should we make it clear?

I asked myself this question when reading.